pkg/fakerp/manifest.go

```diff
@@ -38,7 +38,7 @@ func LoadClusterConfigFromManifest(log *logrus.Entry, manifestTemplate string) (
 	if manifestTemplate == "" {
 		manifestTemplate = "test/manifests/normal/create.yaml"
 	}
-	log.Debugf("generating manifest from %q", manifestTemplate)
+	log.Debugf("generating manifest from %s", manifestTemplate)
```

Comment:
Can you also fix the above message while you are here?

Reply:
Nevermind, I am fixing this in https://github.com/openshift/openshift-azure/pull/897